### DISCUSSION TITLE
PrimeVue DataTable rows per page issue

### DISCUSSION THREAD

**Discussion by fotisskal** [primefaces/community#631] [PrimeVue Templates] [answered]
![Screenshot 2023-12-06 at 10 30 13 PM](https://github.com/primefaces/community/assets/52207059/fbe2da46-1ed3-4626-a2c4-f1ca11c14f00)

As you can see the the number of rows does not appear nicely. I am using Atlantis template with latest primevue. Is there a custom way to fix this or it is a bug? Any help appreciated!

**Comment by tugcekucukoglu** [comment, marked as answer]
Could you try to add `pt` property for the input of the paginator's dropdown

```
DataTable :pt="{
        paginator: {
            rowperpagedropdown: {
                input: {
                    class: 'flex align-items-center'
                }
            }
        }
    }">
```

It's a defect on our end. Thanks for the report.
